contracts/PartyBid.sol

```diff
@@ -154,6 +167,8 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         address _nftContract,
         uint256 _tokenId,
         uint256 _auctionId,
+        address _partyHost,
```

Comment:
after working this in the frontend, i'm backpedalling a little bit on this name.  i think a future (different) feature may be more appropriate for the term 'partyHost'.  it felt more natural to call this `donationAddress` and the bp `donationAmountBasisPoints` in the frontend.  what do you think about doing something similar on the solidity?

Reply:
merged your PR renaming to split